plotly/tools.py

```diff
@@ -3226,9 +3265,9 @@ def _map_face2color(face, colormap, vmin, vmax):
         return face_color
 
     @staticmethod
-    def _trisurf(x, y, z, simplices, colormap=None, color_func=None,
-                 plot_edges=False, x_edge=None, y_edge=None, z_edge=None,
-                 facecolor=None):
+    def _trisurf(x, y, z, simplices, show_colorbar, colormap=None,
```

Comment:
Should we allow `show_colorbar` to be defaulted to `True` or `False`? Seems reasonable to me.